code/datums/spell_targeting/targeted.dm

```diff
@@ -37,12 +37,11 @@
 					for(var/atom/A as anything in possible_targets)
 						if(target)
 							if(get_dist(spell_location, A) < get_dist(spell_location, target))
-								if(spell.los_check(user, A))
-									target = A
-						else
-							if(spell.los_check(user, A))
 								target = A
-			targets += target
+						else
+							target = A
+			if(target)
+				targets += target
```

Comment:
```suggestion
			targets += target
```
Not required anymore. Forgot to remove it after I removed the los_check here.
With the los_check you could have `target` be null. Now you can't